go.mod

```diff
@@ -2,148 +2,126 @@ module github.com/rancher/webhook
-
 replace (
-	k8s.io/api => k8s.io/api v0.27.4
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.27.4
-	k8s.io/apimachinery => k8s.io/apimachinery v0.27.4
-	k8s.io/apiserver => k8s.io/apiserver v0.27.4
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.27.4
-	k8s.io/client-go => github.com/rancher/client-go v1.27.4-rancher1
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.27.4
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.27.4
-	k8s.io/code-generator => k8s.io/code-generator v0.27.4
-	k8s.io/component-base => k8s.io/component-base v0.27.4
-	k8s.io/component-helpers => k8s.io/component-helpers v0.27.4
-	k8s.io/controller-manager => k8s.io/controller-manager v0.27.4
-	k8s.io/cri-api => k8s.io/cri-api v0.27.4
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.27.4
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.27.4
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.27.4
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.27.4
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.27.4
-	k8s.io/kubectl => k8s.io/kubectl v0.27.4
-	k8s.io/kubelet => k8s.io/kubelet v0.27.4
-	k8s.io/kubernetes => k8s.io/kubernetes v1.27.4
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.27.4
-	k8s.io/metrics => k8s.io/metrics v0.27.4
-	k8s.io/mount-utils => k8s.io/mount-utils v0.27.4
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.27.4
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.27.4
+	k8s.io/api => k8s.io/api v0.28.6
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.28.6
+	k8s.io/apimachinery => k8s.io/apimachinery v0.28.6
+	k8s.io/apiserver => k8s.io/apiserver v0.28.6
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.28.6
+	k8s.io/client-go => github.com/rancher/client-go v1.28.6-rancher1
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.28.6
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.28.6
+	k8s.io/code-generator => k8s.io/code-generator v0.28.6
+	k8s.io/component-base => k8s.io/component-base v0.28.6
+	k8s.io/component-helpers => k8s.io/component-helpers v0.28.6
+	k8s.io/controller-manager => k8s.io/controller-manager v0.28.6
+	k8s.io/cri-api => k8s.io/cri-api v0.28.6
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.28.6
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.28.6
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.28.6
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.28.6
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.28.6
+	k8s.io/kubectl => k8s.io/kubectl v0.28.6
+	k8s.io/kubelet => k8s.io/kubelet v0.28.6
+	k8s.io/kubernetes => k8s.io/kubernetes v1.28.6
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.28.6
+	k8s.io/metrics => k8s.io/metrics v0.28.6
+	k8s.io/mount-utils => k8s.io/mount-utils v0.28.6
+	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.28.6
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.28.6
```

Comment:
Anybody knows why we're duplicating most of these entries in `require` and `replace`?

Some of them aren't even imported directly nor indirectly (eg: `k8s.io/cli-runtime`).

If there's no good reason then I can nuke this and only keep `replace` for things that require replace like the client-go fork.